scripts/backend_tests.py

```diff
@@ -230,6 +239,11 @@ def main():
     if parsed_args.test_target and parsed_args.test_path:
         raise Exception('At most one of test_path and test_target '
                         'should be specified.')
+    if parsed_args.test_target and parsed_args.exclude_load_tests:
+        if parsed_args.test_target.endswith('load_test'):
```

Comment:
Maybe this changes to: if test_target starts with _LOAD_TESTS_DIR (you can use os.path operations, I think)